dropshot/tests/fail/bad_endpoint12.stderr

```diff
@@ -1,30 +1,22 @@
-error[E0277]: the trait bound `String: HttpResponse` is not satisfied
+error[E0277]: the trait bound `Result<String, HttpError>: HttpResponse` is not satisfied
   --> tests/fail/bad_endpoint12.rs:15:6
    |
 15 | ) -> Result<String, HttpError> {
-   |      ^^^^^^ the trait `HttpCodedResponse` is not implemented for `String`, which is required by `Result<String, HttpError>: ResultTrait`
+   |      ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `HttpResponse` is not implemented for `Result<String, HttpError>`
```

Comment:
as above; I **think** this might be harder for people to understand? What do you think?